alongtheway.txt

```diff
@@ -620,7 +620,7 @@ Reflect: This is a hard one to reflect on. Notice Peter has made his way into th
 Read: Mark 16:6-7,16:14, Luke 24:9-12, John 21:15-17, Proverbs 3:11-12, Hebrews 12:4-11
 
 Reflect:
-The passage in John is considered the restoration of Peter. Even before then it seemed Peter already knew he was forgiven. Peter was eager to reconcile with Jesus after His resurrection.  When Jesus was telling Peter that he is the rock He will build His church on (Matt 16:18), giving him authority, He knew Peter would deny Him. But he also already knows the love Peter has for Him and the repentant heart he has. Jesus forgives easily and is always after the heart.
+The passage in John is considered the restoration of Peter. Even before then it seemed Peter already knew he was forgiven. Peter was eager to reconcile with Jesus after His resurrection.  When Jesus was giving Peter authority and telling him that he is the rock He will build His church on (Matt 16:18), Jesus knew Peter would deny Him. But He also already knows the love Peter has for Him and the repentant heart he has. Jesus forgives without hesitation and is always after the heart.
```

Comment:
Stamp of approval
On Jan 15, 2016 11:17 PM, "Joshua Maxwell" <notifications@github.com> wrote:

> In alongtheway.txt
> <https://github.com/joshuamaxwell/alongtheway/pull/11#discussion_r49927081>
> :
>
> > @@ -620,7 +620,7 @@ Reflect: This is a hard one to reflect on. Notice Peter has made his way into th
> >  Read: Mark 16:6-7,16:14, Luke 24:9-12, John 21:15-17, Proverbs 3:11-12, Hebrews 12:4-11
> >
> >  Reflect:
> > -The passage in John is considered the restoration of Peter. Even before then it seemed Peter already knew he was forgiven. Peter was eager to reconcile with Jesus after His resurrection.  When Jesus was telling Peter that he is the rock He will build His church on (Matt 16:18), giving him authority, He knew Peter would deny Him. But he also already knows the love Peter has for Him and the repentant heart he has. Jesus forgives easily and is always after the heart.
> > +The passage in John is considered the restoration of Peter. Even before then it seemed Peter already knew he was forgiven. Peter was eager to reconcile with Jesus after His resurrection.  When Jesus was giving Peter authority and telling him that he is the rock He will build His church on (Matt 16:18), Jesus knew Peter would deny Him. But He also already knows the love Peter has for Him and the repentant heart he has. Jesus forgives without hesitation and is always after the heart.
>
> reorder and switch "forgives easily" with "without hesitation"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joshuamaxwell/alongtheway/pull/11/files#r49927081>.
>